include/cpp_bindgen/array_descriptor.h

```diff
@@ -21,7 +21,19 @@ enum gen_fortran_array_kind {
     gen_fk_Float,
     gen_fk_Double,
     gen_fk_LongDouble,
-    gen_fk_SignedChar
+    gen_fk_SignedChar,
+
+#ifdef CPP_BINDGEN_GT_LEGACY // remove once GT is at v2.0
```

Comment:
We should think about how to do such things: Will we have a specific TODO-name for GT 2.0 such that we can grep for it when updating? Should we have a define such that we can do `#error` depending on that macro? Just some ideas

Reply:
I tried to use always this exact string. Additionally we can grep for the macro.